plugins/field-date/README.md

```diff
@@ -49,8 +51,7 @@ Blockly.defineBlocksWithJsonArray([
     }]);
 ```
 
-[View the developer documentation](https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/date) for further usage examples.
+- [View the developer documentation](https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/date) for further usage examples.
```

Comment:
Does this still conform to the documented API? Even after your changes related to date parsing? My reading is that it does, but wanted to check!

Reply:
Looks like the link is no longer available, actually. I'm assuming this was originally in the core repo, though since it's in plugins now it's no longer documented there.

I went ahead and removed this link.